spec/controllers/communicarts_controller_spec.rb

```diff
@@ -157,6 +157,30 @@
         expect(token.used?).to eq(true)
       end
     end
-  end
 
+    context "version number" do
+      it "doesn't need to be present" do
+        put 'approval_response', approval_params_with_token
+        approval.reload
+        expect(approval).to be_approved
+        expect(flash[:error]).not_to be_present
+      end
+      it "works if version matches" do
+        params = approval_params_with_token
+        params[:version] = cart.proposal.version.to_s
```

Comment:
Minor: if you still like the idea of [removing the Cart model](https://www.pivotaltracker.com/story/show/91211506), might be good for our specs to not reference them if they don't need. Mind putting a `let(:proposal) { cart.proposal }` above and changing this and the following specs?